index.d.ts

```diff
@@ -300,9 +300,6 @@ export class FieldBase extends ReflectionObject {
      */
     constructor(name: string, id: number, type: string, rule?: (string|{ [k: string]: any }), extend?: (string|{ [k: string]: any }), options?: { [k: string]: any }, comment?: string);
 
-    /** Field rule, if any. */
-    public rule?: string;
```

Comment:
Not sure why the .d.ts file is changed in this PR, is it intended?

Reply:
Seems to be a consequence of running `build` again, I suspect its been changed in some other commit but the project hasn't been fully rebuilt since?